plugins/archlinux/archlinux.plugin.zsh

```diff
@@ -182,3 +182,25 @@ if (( $+commands[yay] )); then
   alias yaupd="yay -Sy"
   alias upgrade='yay -Syu'
 fi
+
+
+if (( $+commands[paru] )); then
+  alias paclean='paru -Sc'
+  alias paclr='paru -Scc'
+  alias paupg='paru -Syu'
+  alias pasu='paru -Syu --noconfirm'
+  alias pain='paru -S'
+  alias pains='paru -U'
+  alias pare='paru -R'
+  alias parem='paru -Rns'
+  alias parep='paru -Si'
+  alias pareps='paru -Ss'
+  alias paloc='paru -Qi'
+  alias palocs='paru -Qs'
+  alias palst='paru -Qe'
+  alias paorph='paru -Qtd'
+  alias painsd='paru -S --asdeps'
+  alias pamir='paru -Syy'
+  alias paupd="paru -Sy"
+  alias upgrade='paru -Syu'
+fi
```

Comment:
nit: keep new line at the end, please.  most editors insert it on save anyway, so would just clutter git diffs later.